deps/chakrashim/src/inspector/v8-console.cc

```diff
@@ -839,11 +839,9 @@ V8Console::CommandLineAPIScope::~CommandLineAPIScope() {
     v8::Local<v8::Value> name;
     if (!names->Get(m_context, i).ToLocal(&name) || !name->IsName()) continue;
     if (name->IsString()) {
-      v8::Local<v8::Value> descriptor;
-      bool success = m_global
-                         ->GetOwnPropertyDescriptor(
-                             m_context, v8::Local<v8::Name>::Cast(name))
-                         .ToLocal(&descriptor);
+      v8::Local<v8::Value> value;
+      bool success = m_global->Get(m_context, v8::Local<v8::Name>::Cast(name))
```

Comment:
What is this actually doing? Getting some named property from the global object, but then not using the resulting `value`?

Reply:
Spoke offline; we should definitely add a comment here explaining that the getter for these values is self-deleting based on `m_cleanup`.

Reply:
Yup, already modified it and just verifying some other changes before I push the next set.